server/model/MatchModel.js

```diff
@@ -0,0 +1,14 @@
+export class MatchModel {
```

Comment:
Was the goal of this ticket to create a class or a MongoDB model? If this data is to be saved to the DB it may be better to just declare a Schema and a Model and skip declaring a class entirely.

Additionally you'll want to use module.exports on the server side